Gruntfile.js

```diff
@@ -48,6 +48,13 @@ module.exports = function(grunt) {
                 dest: 'build/<%= pkg.name %>.js'
             },
         },
+        copy: {
+            files: {
+                cwd: '.',
+                src: 'scripts/ableplayer.d.ts',
+                dest: 'build/ableplayer.d.ts'
+            }
+        },
```

Comment:
I've put the file into `scripts` dir to be next to ableplayer-base.js, so had to have a grunt target to move it to `build` dir.